src/store-api/src/region_request.rs

```diff
@@ -389,6 +392,8 @@ pub enum AlterKind {
         /// Columns to change.
         columns: Vec<ChangeColumnType>,
     },
+    /// Change table options.
+    ChangeTableOptions { options: Vec<ChangeTableOption> },
```

Comment:
This is region level.
```suggestion
    /// Change region options.
    ChangeRegionOptions { options: Vec<ChangeRegionOption> },
```

Reply:
> ChangeTableOptions { options: Vec<Chan

fixed in https://github.com/GreptimeTeam/greptimedb/commit/dce487e66e70eb38617be606b3fcbbb74b809111#diff-839e23ed322c90eacc42fdddeb1f7108335e6434eba0a7a1c6dbc42052a8b9e9R396